support/reftest/reftest

```diff
@@ -196,8 +196,8 @@ def prepare(db, s3, config, opt):
             url, opt.key, opt.cert, opt.cacert
         )
 
-        # checksum verify
-        if cdn_data_checksum != item["sha256"]:
+        # For unstable content which did not provide sha256, it will skip the checksum verify
+        if item["sha256"] and cdn_data_checksum != item["sha256"]:
```

Comment:
This is OK, but I think it's a little more friendly to make sha256 optional rather than forcing a blank entry in the file.  That could be done by using item.get('sha256') here.